tools/hive/hadoop-hive/scripts/hadoop-bootstrap.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -x
+
+# used by Hadoop
+/usr/sbin/sshd
+
+: ${HADOOP_PREFIX:=/usr/local/hadoop}
+
+$HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+
+rm /tmp/*.pid
+
+# templating of config files
+sed s/{{HOSTNAME}}/$HOSTNAME/ /usr/local/hadoop/etc/hadoop/core-site.xml.template > /usr/local/hadoop/etc/hadoop/core-site.xml
+sed s/{{HOSTNAME}}/$HOSTNAME/ /usr/local/hadoop/etc/hadoop/yarn-site.xml.template > /usr/local/hadoop/etc/hadoop/yarn-site.xml
```

Comment:
Follow up to my question below, it doesn't look like `yarn-site.xml.template` has a `{{HOSTNAME}}` space to fill

Reply:
I'll admit this one is a little confusing. What happens is this script gets called in both the non-kerberized and kerberized images. And, the kerberized version of `yarn-site.xml` does have a `{{HOSTNAME}}` in it, and I wanted to avoid special processing to account for that.